gatsby-config.js

```diff
@@ -72,6 +72,7 @@ module.exports = {
         theme_color: `#f5f0eb`,
         start_url: `/`,
         display: `standalone`,
+        icon: require.resolve('./src/images/favicon.png'),
```

Comment:
I removed the house favicon, in favour of a simple SVG icon: 🏠
So, please don't re-add it.

Reply:
ok, will remove the entry in the config and the image file but I don't see the 🏠 being used in the live version

![image](https://user-images.githubusercontent.com/9373787/77487151-9625de00-6e29-11ea-8e2a-b93a860f0cd1.png)

Maybe redeploy with cache clear on netlify 🤔

Reply:
Done https://github.com/homeferences/website/pull/10/commits/f548d61a5706d6b049349bee0c196f149f81b208